src/Classes/Api.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace ListingsApp\Classes;
+/**
+ * send api request to receive listings data
+ */
+class Api {
+    private $request;
+    private array $allListings;
+    private array $lettingTypes;
+    private array $lettingStatuses;
+
+    /**
+     * Api constructor.
+     */
+    public function __construct() {
+        $this->request = curl_init();
+        var_dump($this->request);
```

Comment:
Is this var_dump functional or can it go?

Reply:
it can go, thx